doc/source/examples/resolved_cfd-dem/sphere-carreau-with-sharp-interface/sphere-carreau-with-sharp-interface.rst

```diff
@@ -0,0 +1,269 @@
+
+.. code-block:: text
+
+	subsection boundary conditions
+	  set number                  = 5
+	  subsection bc 0
+		set id 		= 0
+		set type    = function
+		subsection u
+		  set Function expression = 1
+		end
+		subsection v
+		  set Function expression = 0
+		end
+		subsection w
+		  set Function expression = 0
+		end
+	  end
+	  subsection bc 1
+		set id 		= 2
+		set type    = slip
+	  end    
+	  subsection bc 2
+		set id 		= 3
+		set type    = slip
+	  end
+	  subsection bc 3
+		set id 		= 4
+		set type    = slip
+	  end
+	  subsection bc 4
+		set id 		= 5
+		set type    = slip
+	end
+	end
+
+Physical properties
+~~~~~~~~~~~~~~~~~~~~
+
+This example showcases a shear-thinning flow, for which the viscosity decreases when the local shear rate increases. The Carreau model is being used. For more information on rheological models, see :doc:`../../../parameters/cfd/physical_properties`
+
+.. code-block:: text
+
+	subsection physical properties
+	  set number of fluids = 1
+	  subsection fluid 0
+		set rheological model	= carreau
+		subsection non newtonian
+		  subsection carreau
+			set n 		   	= 0.5
+			set viscosity_0    	= 0.063403
+			set viscosity_inf  	= 0
+			set lambda	   	= 10
+			set a	           	= 2.0
+		  end
+		end
+	  end
+	end
+
+With ``viscosity_inf = 0`` (3-parameter Carreau model), the *a priori* Reynolds number can be estimated using :
+
+.. math::
+
+	 Re = \frac{u_{\infty}D(1+(\lambda(\frac{u_\infty}{D}))^2)^{\frac{1-n}{2}}}{\eta_0}
+
+We use an *a priori* Reynolds number, since it is not possible, *a priori*, to know the effective viscosity of the flow. For the given parameters, the *a priori* Reynolds number is :math:`50`. 
+
+Initial conditions
+~~~~~~~~~~~~~~~~~~~~
+
+This example uses a ramping initial condition that first ramps on the ``n`` parameter, and then on the ``viscosity_0`` parameter. This allows for a smooth transition of non-Newtonian behavior level and of regime.
+
+.. code-block:: text
+
+	subsection initial conditions
+	  set type = ramp
+	  subsection ramp
+		subsection n
+		  set initial n 	= 1.0
+		  set iterations 	= 2
+		  set alpha 		= 0.5
+		end
+		subsection viscosity
+		  set initial viscosity = 1.0
+		  set iterations 	= 2
+		  set alpha 		= 0.5
+		end
+	  end
+	end
+	
+The first initial condition simulation solves for ``n=1.0``, ``viscosity_0 = 1.0``, ``viscosity_inf = 0``, ``lambda=10`` and ``a=2``. The subsequent initial simulations are:
+
+* (Second ``n`` iteration) ``n=0.75``, ``viscosity_0 = 1.0``, ``viscosity_inf = 0``, ``lambda=10`` and ``a=2`` ;
+* (First ``viscosity`` iteration) ``n=0.5``, ``viscosity_0 = 1.0``, ``viscosity_inf = 0``, ``lambda=10`` and ``a=2`` ;
+* (Second ``viscosity`` iteration) ``n=0.5``, ``viscosity_0 = 0.531702``, ``viscosity_inf = 0``, ``lambda=10`` and ``a=2`` 
```

Comment:
I would explain here what is the meaning of the iterations parameters so that people understand the origin of the 2 :)

Reply:
The equation is in the initial conditions documentation, should it also be added here?

Reply:
Nah I think it's ok.